api/src/main/java/uk/gov/dhsc/htbhf/claimant/model/EligibilityOverrideDTO.java

```diff
@@ -32,6 +33,10 @@
     @ApiModelProperty(notes = "The date at which the override expires", example = "2025-05-17")
     private LocalDate overrideUntil;
 
+    @JsonProperty("qualifyingBenefits")
+    @ApiModelProperty(notes = "Overrides the reason that this applicant qualifies for Healthy Start", example = "under_18")
```

Comment:
I think this description really confuses matters because I would expect this to be their qualifying benefit not their reason for overriding. I understand why this has been done but it feels like a mistake to me, this enum now has two means and purposes and this API description highlights the confusion very well.

Reply:
Would the confusion be resolved if we replaced `QualifyingBenefits` in the CombinedIdentityAndEligibilityResponse with a new `QualifyingReason` enum that has the same values as `QualifyingBenefit` plus `under_18`?

Reply:
Yes, that might work.

Reply:
Its bit confusing because QualifyingBenfits are returned from DWP, but now we are also using this as a qualifying reason. Can we create a new enum as matt suggested but with just under_18 and not_set?

Reply:
will keep it as is for now and will refactor in future if it's more confusing.

Reply:
I really think the change should be done now, I see no reason to delay work for the future if several people have said it's confusing.